hardware/optical_power_meters.py

```diff
@@ -18,7 +18,27 @@
 """
 
 import visa
+from hardware import u
 
+class MockOpticalPowerMeter:
+
+     def __init__(self, instr_name = None):
+        if not instr_name:
+            self.name = "Optical Power Meter - Newport 1830C"
+        else:
+            self.name = instr_name
+        self._power = 5 * u.watt
+
+    def identify(self):
+        return self.name
+
+    @property
+    def power(self):
+        return self._power
+
+    @power.setter

```

Comment:
Most meters do not have a "set power" setting, in the same way you wouldn't set a voltage in a volt meter. We can drop `@power.setter`